inhouse/command_handlers/queue.py

```diff
@@ -164,7 +164,8 @@ async def attempt_complete_match(self, message_id, winner, main_leaderboard: Lea
 
         if main_leaderboard != None:
```

Comment:
So correct me if I am wrong, but its going to fall into this if if main_leaderboard is set. So what I think will happen is it will update the leaderboard for casual matches if main_leaderboard is set (if that makes sense). Also the rest of everything looks good to me.

Reply:
Nice catch. Fixed.